plugins/c8y_log_plugin/src/main.rs

```diff
@@ -186,27 +182,6 @@ fn init(config_dir: &Path, logs_dir: &Path) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
-/// append the log plugin file with software-management logs
-/// assumes file is already created.
-fn create_default_log_plugin_file(path_to_toml: &str, logs_dir: &str) -> Result<(), anyhow::Error> {
-    let logs_path = format!("{logs_dir}/tedge/agent/software-*");
-    let data = toml::toml! {
-        files = [
-            { type = "software-management", path = logs_path }
-        ]
-    };
-
-    let mut toml_file = OpenOptions::new()
-        .append(true)
-        .create(false)
-        .open(path_to_toml)
-        .map_err(|error| {
-            anyhow::anyhow!("Unable to open file: {}. Error: {}", path_to_toml, error)
-        })?;
-    toml_file.write_all(data.to_string().as_bytes())?;
-    Ok(())
-}
-
 /// for the log plugin to work the following directories and files are needed:
 ///
 /// Directories:
```

Comment:
It's not related to your PR, but the comments below contains inaccurate.

Wrong:
```
/// - CONFIG_DIR/c8y/log/c8y-log-plugin.toml
```

Correct:
```
/// - CONFIG_DIR/c8y/c8y-log-plugin.toml
```

Reply:
done